roadmap/4_governance_nns/upcoming/1_nns_large_number_of_neurons.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Support large number of neurons
```

Comment:
I wonder a bit whether we describe an attack vector here and should rather not disclose this. 
WDYT @andrew-lee-work ?